### ISSUE TITLE
‘arg0’ was not declared in this scope

### ISSUE THREAD

**Issue by ReneHollander** [bytedeco/javacpp#99]
The generated C++ code sometimes wants to use an argument `arg0` that is not declared in the function protoype. I assume that the issue is on my end, because it worked before. I didn't work on the project for a few months and I can't recall if it actually worked or how to fix it. The provided code is only from one of the methods that wants the `arg0` argument. There are many more that aren't declared correctly.
But I am also getting the following warnings from the Generator:
```
Warning: First parameter of caller method call() or apply() for member function pointer org.duktape.duktapejava.Duktape.Function is not a Pointer. Compilation will most likely fail.
Warning: First parameter of caller method call() or apply() for member function pointer org.duktape.duktapejava.Duktape.FatalHandler is not a Pointer. Compilation will most likely fail.
Warning: First parameter of caller method call() or apply() for member function pointer org.duktape.duktapejava.Duktape.Function is not a Pointer. Compilation will most likely fail.
Warning: First parameter of caller method call() or apply() for member function pointer org.duktape.duktapejava.Duktape.FatalHandler is not a Pointer. Compilation will most likely fail.
```
I debugged the Generator a little, and all the warnings are caused by the allocate function not having an argument that is a Pointer.

GCC output:
```
/home/rene/Dropbox/Code/duktape-java/build/tmp/native/duktape-java-linux-x86_64.cpp: In function ‘int JavaCPP_org_duktape_duktapejava_Duktape_00024Function_allocate_callback()’:
/home/rene/Dropbox/Code/duktape-java/build/tmp/native/duktape-java-linux-x86_64.cpp:1384:93: error: ‘arg0’ was not declared in this scope
     return JavaCPP_org_duktape_duktapejava_Duktape_00024Function_allocate_callback_instance(arg0);
                                                                                             ^
```
Snipped from the generated C++ code:
```c++
JNIEXPORT int JavaCPP_org_duktape_duktapejava_Duktape_00024Function_allocate_callback() {
    return JavaCPP_org_duktape_duktapejava_Duktape_00024Function_allocate_callback_instance(arg0);
}
```
The used Java definition:
```java
@org.bytedeco.javacpp.annotation.Namespace("")
public static abstract class Function extends org.bytedeco.javacpp.FunctionPointer {

    static {
        LibraryLoader.load();
    }

    public Function() {
        allocate();
    }

    private native void allocate();

    public abstract int call(duk_context ctx);

}
```
```java
@Namespace("")
@Name("duk_context")
public class duk_context
    extends Pointer
{

    static { org.duktape.duktapejava.LibraryLoader.load(); }
}
```

**Comment by saudet** [comment]
Fixed in the latest commit above. Thanks for reporting!

**Comment by ReneHollander** [comment]
Are you planning to release a patch? Setting the Namespace to "" is mandatory if someone wants to generate bindings for C code.

**Comment by saudet** [comment]
Fix included in version 1.2.1!
